app/controllers/notifications_controller.rb

```diff
@@ -9,4 +9,38 @@ def index
       format.js
     end
   end
+
+  def update
+    @notifications =
+      if params[:id] == 'all'
+        current_user.notifications
+      elsif params[:id]
+        current_user.notifications.where(id: params[:id])
+      end
+
+    @notifications.each(&:read!)
```

Comment:
If we have 10 notifications this is going to cause 10 separate `UPDATE` calls. Is that what we want?